starter_code/app.js

```diff
@@ -2,20 +2,40 @@
 const express = require('express');
 const hbs     = require('hbs');
 const app     = express();
+
```

Comment:
Try to put all the requires at the top fo the file, separated from the rest. It's cleaner.